import_tables.sql

```diff
@@ -43,19 +27,6 @@ SELECT
 FROM pluto
 INNER JOIN borough ON pluto.borough_id=borough.abbr;
 
-BEGIN;
```

Comment:
Same as above, I think we should still keep this until we have a new PR that does the data transform and load from the source data tables